azure-pipelines.sds.daily.yml

```diff
@@ -3,8 +3,8 @@ trigger: none
 pr: none
 
 schedules:
-  - cron: "0 0 * * *"
-    displayName: Daily midnight build
+  - cron: "0 8 * * *"
```

Comment:
This is UTC, so will be 9am BST - is that intended?

Reply:
Thanks! I forgot about the timezone. I did think that at first but the whole suite runs in about 10 mins. But I actually we probably should aim for as early as possible to minimise the change needed when the tests grow